js/editmode.js

```diff
@@ -28,7 +28,7 @@ function leave() {
 	var k = 'settings:Edit Manually';
 	$('.go_editmode_button').text(state._(k)).attr('data-i18n', k);
 	hide_inputs(0);
-	$('#score td.score span').show();
+	$('#score').find('td.score span').show();
```

Comment:
Even if this is faster with jQuery, it's quite ugly. In many places (this one included) we can just use the `uiu` helper functions which do away with all the jQuery overhead.